docs/tutorials/multimodal/object_detection/index.rst

```diff
@@ -30,6 +30,18 @@ Quick Start
 
       How to train high quality object detection model with MultiModalPredictor in under 5 minutes on COCO format dataset.
 
+Pre-requisite
```

Comment:
There is already per-requisite before the quick start tutorial, maybe we don't need to add it again here.

Reply:
Oh this was probably from rebasing. I'll delete this.